modules/rgbd/include/opencv2/rgbd/volume.hpp

```diff
@@ -50,9 +49,66 @@ enum class VolumeType
     HASHTSDF = 1
 };
 
-CV_EXPORTS_W cv::Ptr<Volume> makeVolume(VolumeType _volumeType, float _voxelSize, cv::Matx44f _pose,
-                           float _raycastStepFactor, float _truncDist, int _maxWeight,
-                           float _truncateThreshold, Vec3i _resolution);
+struct CV_EXPORTS_W VolumeParams
+{
+    /** @brief Type of Volume
+        Values can be TSDF (single volume) or HASHTSDF (hashtable of volume units)
+    */
+    VolumeType type;
+
+    /** @brief Resolution of voxel space
+        Number of voxels in each dimension.
+    */
+    Vec3i resolution;
+
+    /** @brief Resolution of volumeUnit in voxel space
+        Number of voxels in each dimension for volumeUnit
+        Applicable only for hashTSDF.
+    */
+    int unitResolution = 0;
+
+    /** @brief Initial pose of the volume in meters */
+    Affine3f pose;
+
+    /** @brief Length of voxels in meters */
+    float voxelSize;
+
+    /** @brief TSDF truncation distance
+        Distances greater than value from surface will be truncated to 1.0
+    */
+    float tsdfTruncDist;
+
+    /** @brief Max number of frames to integrate per voxel
+        Each voxel stops integration after the maxWeight is crossed
```

Comment:
A voxel does not stop integration, it works like a running average across last `maxWeight` frames. Need to rephrase this somehow.

Reply:
Done.